hw/cxl/cxl-host.c

```diff
@@ -210,36 +209,36 @@ static MemTxResult cxl_read_cfmws(void *opaque, hwaddr addr, uint64_t *data,
 
     d = cxl_cfmws_find_device(fw, addr);
     if (d == NULL) {
+        trace_cxl_debug_message("CXL device not found");
         *data = 0;
         /* Reads to invalid address return poison */
         return result;
     }
-    type = object_get_typename(OBJECT(d));
 
-    g_assert(addr < 0x8000000);
```

Comment:
Why remove assert?

Reply:
Not sure why these two lines disappeared - added back.